sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -123,6 +124,73 @@ static inline void gnrc_sixlowpan_ctx_remove(uint8_t id)
 {
     gnrc_sixlowpan_ctx_lookup_id(id)->prefix_len = 0;
 }
+
+/**
+ * @brief   Check if a prefix matches a compression context
+ *
+ * @param[in] ctx        The compression context
+ * @param[in] prefix     IPv6 prefix
+ * @param[in] prefix_len Length of the IPv6 prefix
+ *
+ * @return    true if the prefix matches the compression context.
+ */
+static inline bool gnrc_sixlowpan_ctx_match(const gnrc_sixlowpan_ctx_t *ctx,
+                                            const ipv6_addr_t *prefix, uint8_t prefix_len)
+{
+    return (ctx != NULL) &&
+           (ctx->prefix_len == prefix_len) &&
+           (ipv6_addr_match_prefix(&ctx->prefix, prefix) >= prefix_len);
+}
```

Comment:
As this function does not use any functions of the `gnrc_sixlowpan_ctx` module, it should be fine if it is not protected by an `IS_USED(MODULE_GNRC_SIXLOWPAN_CTX)` conditional.